recipe/meta.yaml

```diff
@@ -8,13 +8,11 @@ package:
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/websocket-client-{{ version }}.tar.gz
-  sha256: 6278a75065395418283f887de7c3beafb3aa68dada5cacbe4b214e8d26da499b
+  sha256: 5051b38a2f4c27fbd7ca077ebb23ec6965a626ded5a95637f36be1b35b6c4f81
 
 build:
   number: 0
   noarch: python
```

Comment:
```suggestion
```

This needs to be removed because of the `scripts` tag: https://github.com/websocket-client/websocket-client/blob/b5ec9e9728d549258d9b9183c1857489bdf5c07f/setup.py#L67